src/app/(proper_react)/components/ControlsWrapper.module.scss

```diff
@@ -0,0 +1,142 @@
+@import "../../tokens";
+
+.wrapper {
+  display: flex;
+  height: 100%;
+}
+
+.mainMenu {
+  flex: 0 0 250px;
+  box-shadow: 0px 4px 12px 0px $color-grey-10;
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+  padding: 1px;
```

Comment:
Nit: The padding prevents the sidebar from collapsing entirely and has still a width of `2px` when closed.

Reply:
Oh, good point. I just encountered a different Figma where the sidebar isn't even collapsible in the first place, so I'd propose not spending too much effort refining the styling until we have actual designs for it.